### PR TITLE
Add new KB - Creating TableCells with GoToAction and UriAction

### PR DESCRIPTION
**Pull request by mariaterzieva12**
Add new KB - Creating TableCells with GoToAction and UriAction

### CODE REVIEW

File: knowledge-base/creating-tablecells-with-gotoaction-and-uriaction.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+---
+title: Creating TableCells with GoToAction and UriAction
+description: This article demonstrates how can create table cells with GoTo action and UriAction in PDF document with Telerik Document Processing.
+type: how-to 
+page_title: Creating TableCells with GoToAction and UriAction
+slug:
```
Comment (DKaramfilov) [inline knowledge-base/creating-tablecells-with-gotoaction-and-uriaction.md:6]: No Slug!

File: knowledge-base/creating-tablecells-with-gotoaction-and-uriaction.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+res_type: kb
+---
+
+|Product Version|Product|Author|
+|----|----|----|
+|N/A|Telerik Document Processing|Maria Terzieva|
+
+## Description
+
+The pdf format is different than the format in MS Word or other commonly used editors. With the PDF format, each content element has a specific position, and in some cases, each character is positioned separately. PDF standard does not have any information about tables as well, once exported the tables in the document are represented by lines and text fragments.
```
Comment (DKaramfilov) [inline knowledge-base/creating-tablecells-with-gotoaction-and-uriaction.md:18]: The first sentence does bring any value, can be removed. extend te description a bit more and describe the proposed solution.

File: knowledge-base/creating-tablecells-with-gotoaction-and-uriaction.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+## Description
+
+The pdf format is different than the format in MS Word or other commonly used editors. With the PDF format, each content element has a specific position, and in some cases, each character is positioned separately. PDF standard does not have any information about tables as well, once exported the tables in the document are represented by lines and text fragments. 
+
+Although the editors do not allow you to insert the widgets it is possible to insert a place holder and then use its postion to insert the widget.
+
+## Solution
+
+This article shows example of how you implement solution for inserting a widget.
```
Comment (DKaramfilov) [inline knowledge-base/creating-tablecells-with-gotoaction-and-uriaction.md:24]: The widget itself is not the purpose of the article in this case.

File: knowledge-base/creating-tablecells-with-gotoaction-and-uriaction.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+
+## Table of Contents
```
Comment (DKaramfilov) [inline knowledge-base/creating-tablecells-with-gotoaction-and-uriaction.md:26]: We do not need TOC

File: knowledge-base/creating-tablecells-with-gotoaction-and-uriaction.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+- [Convert a Spreadsheet Document to PDF](#convert-a-spreadsheet-document-to-pdf)
+	- [XLSX to PDF](#convert-xlsx-to-pdf)
+	- [XLS to PDF](#convert-xls-to-pdf)
+	- [CSV to PDF](#convert-csv-to-pdf)	
+	- [DataTable object to PDF](#convert-datatable-to-pdf)	
+	
+
+## Insert widgets in table cell
+
+````C#
```
Comment (DKaramfilov) [inline knowledge-base/creating-tablecells-with-gotoaction-and-uriaction.md:43]: No regions for code. See other articles

**Review by DKaramfilov — CHANGES_REQUESTED** [review]
It apers to be unfinished. Did you forget to save it?

**Comment by mariaterzieva12** [comment]
The comments are resolved with the new commit.

File: knowledge-base/creating-tablecells-with-gotoaction-and-uriaction.md [hunk 6]
```diff
@@ -0,0 +1,110 @@
+---
+title: Creating TableCells with GoToAction and UriAction
+description: This article demonstrates how can create table cells with GoTo action and UriAction in PDF document with Telerik Document Processing.
+type: how-to 
+page_title: Creating TableCells with GoToAction and UriAction
+slug: kb-create-table-cells
+position: 0
+tags: pdf, datatable, cells, widget
```
Comment (TanyaDimitrova) [inline knowledge-base/creating-tablecells-with-gotoaction-and-uriaction.md:8]: datatable and widget are not appropriate tags for this topic

File: knowledge-base/creating-tablecells-with-gotoaction-and-uriaction.md [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+
+|Product Version|Product|Author|
+|----|----|----|
+|N/A|RadPdfProcessing|Maria Terzieva|
+
+## Description
+
+With the PDF format, each content element has a specific position, and in some cases, each character is positioned separately. PDF standard does not have any information about tables as well, once exported the tables in the document are represented by lines and text fragments. 
+
+Although the editors do not allow you to insert the widgets it is possible to insert a place holder and then use its postion to insert the widget.
```
Comment (TanyaDimitrova) [inline knowledge-base/creating-tablecells-with-gotoaction-and-uriaction.md:20]: This line doesn't seem related to the topic

**Review by TanyaDimitrova — CHANGES_REQUESTED** [review]
It would be nice to include the component name first thing in the .md file name. This way, it would be easier to visually see at least the component in the list of KB files.

File: knowledge-base/creating-tablecells-with-gotoaction-and-uriaction.md [hunk 8]
```diff
@@ -0,0 +1,110 @@
+---
+title: Creating TableCells with GoToAction and UriAction
+description: This article demonstrates how can create table cells with GoTo action and UriAction in PDF document with Telerik Document Processing.
```
[the remark this answers is not in the document]
Comment (mariaterzieva12) [reply knowledge-base/creating-tablecells-with-gotoaction-and-uriaction.md:3]: Done

**Comment by mariaterzieva12** [comment]
I have updated the requested changes.

File: knowledge-base/pdfprocessing-creating-tablecells-with-gotoaction-and-uriaction.md [hunk 9]
```diff
@@ -0,0 +1,115 @@
...
+---
+
+|Product Version|Product|Author|
+|----|----|----|
+|N/A|RadPdfProcessing|Maria Terzieva|
+
+## Description
+
+With the PDF format, each content element has a specific position, and in some cases, each character is positioned separately. PDF standard does not have any information about tables as well, once exported the tables in the document are represented by lines and text fragments. 
+For this reason we use Action which defines the behaviour for an annotation. And annotation element associates an object with a location on a RadFixedPage.
```
Comment (TanyaDimitrova) [inline knowledge-base/pdfprocessing-creating-tablecells-with-gotoaction-and-uriaction.md:19]: I am still not sure I can understand why actions are needed and what this example shows from the description.

**Review by TanyaDimitrova — CHANGES_REQUESTED** [review]
There are two identical files - one with the old name and one with the new one

**Comment by mariaterzieva12** [comment]
the file with the old name is deleted

**Pull request merged**